nodes/sample_host.json

```diff
@@ -1,5 +1,5 @@
 {
-  "run_list":["role[mysql]","role[rails_passenger]", "role[sysadmins]"],
+  "run_list":["role[mysql]","role[remove_applications]", "role[rails_passenger]", "role[sysadmins]"],
```

Comment:
True. Since we already have all the other non-required rules in here: let's just use this sample file as a "dump all" for all features :)